docs/config/README.md

```diff
@@ -850,7 +850,7 @@ The module will be shown if any of the following conditions are met:
 
 | Option     | Default                            | Description                                     |
 | ---------- | ---------------------------------- | ----------------------------------------------- |
-| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                      |
+| `format`   | `"via [$symbol$version ]($style)"` | The format for the module.                      |
```

Comment:
This needs to be changed to
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                      |
```
So that we don't end up with too many spaces after the `$symbol` if the version can't be found.